products/idn/api/patch-requests.md

```diff
@@ -0,0 +1,500 @@
+| remove | Removes a value from the target location. The target location must exist for the operation to be successful. | This example uses the [Patch Source](https://developer.sailpoint.com/idn/api/v3/update-source) endpoint to remove an existing filter string from a source's connector: ``` [{"op": "remove", "path": "/connectorAttributes/filterString", "value": "!( id.contains( \"m\" ) )"}] ``` | 
+| replace | Replaces the value at the target location with a new value. The operation object must contain a "value" member whose content specifies the replacement value, and the target location must exist for the operation to be successful. This operation is the equivalent of a "remove" followed by an "add". | This example uses the [Patch Source](https://developer.sailpoint.com/idn/api/v3/update-source) endpoint to replace a source's existing features with new ones: ``` [{"op": "replace", "path": "/features", "value": ["PASSWORD", "PROVISIONING", "ENABLE", "AUTHENTICATE"]}] ``` |
+| move | Removes the operation from a specified location and adds it to the target location. This operation object must contain a "from" member whose content specifies the location to remove the value from, and the "from" location must exist for the operation to be successful. | This example uses the [Patch Source Schema](https://developer.sailpoint.com/idn/api/v3/update-source-schema) endpoint to move an attribute from the beginning to the end of the schema's array of attributes: ``` [{"op": "move", "from": "/attributes/0", "path": "/attributes/-"}] ``` |
+| copy | Copies the value from a specified location to the target location. The operation object must contain a "from" member whose content specifies the location to copy the value from, and the "from" location must exist for the operation to be successful. | This example uses the [Patch Source Schema](https://developer.sailpoint.com/idn/api/v3/update-source-schema) endpoint to copies an attribute from the beginning and duplicates it at the end of the schema's array of attributes: ``` [{"op": "copy", "from": "/attributes/0", "path": "/attributes/-"}] ```  |
+| test | Tests that a value at the target location is equal to a specified value. The operation object must contain a "value" member whose content specifies the value to be compared to the target location's value, and the values must be equal for the operation to be successful. For more information about what "equal" means for different JSON types, refer [here](https://datatracker.ietf.org/doc/html/rfc6902#section-4.6). | This example uses the [Patch Source](https://developer.sailpoint.com/idn/api/v3/update-source) endpoint to test a source's existing features to make sure they match the specified values.: ``` [{"op": "test", "path": "/features", "value": ["PASSWORD", "PROVISIONING", "ENABLE", "AUTHENTICATE"]}] ``` |
```

Comment:
I'll explain it more - as I understand it, "test" basically tests to make sure that the values match the values you provide. It might be more complicated, but I didn't see much more about it in the documentation. I'll ask my teammates to get some more info about why/when someone would want to use it. I guess you might want to check that something is enabled, for example, test that value, and see easily without looking through all the details. Then you could patch the value to update it.